visual-js/visual-playwright/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM node:18 AS runner
+
+WORKDIR app
+
+RUN corepack enable
+
+COPY . ./
+
+RUN yarn install && npm run build --workspaces --if-present
+
+COPY ./visual-playwright/integration-tests/ ./integration-tests/
```

Comment:
Isn't this useless ?

Hasn't `COPY . ./` already pushed that folder too ?

Reply:
True... But it pushes it into a nested folder. Let me see how it behaves if I remove that line.